app/uk/gov/hmrc/servicecommissioningstatus/service/StatusCheckService.scala

```diff
@@ -44,20 +45,25 @@ class StatusCheckService @Inject()(
   import Check.{SimpleCheck, EnvCheck}
   def commissioningStatusChecks(serviceName: String)(implicit hc: HeaderCarrier): Future[List[Check]] =
     for {
+      oRepo           <- teamsAndReposConnector.findRepo(serviceName)
+      isFrontend      =  oRepo.map(_.serviceType) == Some(TeamsAndRepositoriesConnector.ServiceType.Frontend)
```

Comment:
This comparison looks odd - can we use exists?

Reply:
yeah sure